requirements/edx/github.in

```diff
@@ -78,8 +78,10 @@
 # Release candidates being tested.
 ##############################################################################
 
-# ... add dependencies here
-
+# https://github.com/openedx/django-pyfs/pull/169
+git+https://github.com/open-craft/django-pyfs.git@opencraft-release/3.5.0#egg=openedx-django-pyfs==3.5.0
+# https://github.com/openedx/RecommenderXBlock/pull/75
```

Comment:
Sorry I didn't mention this, but this is added via Grove, we don't need it in all our instances